common.go

```diff
@@ -71,10 +71,13 @@ func FormatErrors(errs []OpsLevelErrors) error {
 	var errstrings []string
 	errstrings = append(errstrings, "OpsLevel API Errors:")
 	for _, err := range errs {
-		errstrings = append(errstrings, fmt.Sprintf("\t* %s", string(err.Message)))
+		if len(err.Path) > 0 && err.Path[0] == "base" {
```

Comment:
I would check `len() == 1` here, just in case we end up with sub-elements nested under `base` that have semantic value at some point? But it probably doesn't matter.

Reply:
good to know